### PR TITLE
batch: two gates that suppressed the work instead of the defect, and the guard the batch found unwired

### PR DESCRIPTION
**Pull request by noahgift**
Three sub-tickets for APR-PERF-GATE-001 (#2706). Two of them fix **gates whose failure mode was to suppress the work rather than the defect**; the third is a batch of guards from earlier in the epic.

## PERF-051 — the complexity gate froze the files most worth editing (#2766)

The pre-commit gate scanned **staged files** with an **absolute** threshold, so a file carrying a pre-existing violation could not be edited at all. During #2753 this blocked **four** files, each time forcing instrumentation somewhere less apt:

```
gemm.rs           cublas_prefill_gemm   Cognitive 30
forward_utils.rs  debug_q6k_lm_head     Cognitive 41, reached through include!
gemv_dispatch.rs  three violations, two of them in OTHER files
apply.rs          apply_qkv_bias        Cognitive 31
```

**The gate was not where the tracked `.githooks/` suggests.** `core.hooksPath = .git/hooks-apr`, which execs a pmat-generated `.git/hooks/pre-commit` marked "DO NOT EDIT" and untracked. The tracked `.githooks/pre-commit` is dead in this checkout and never had a complexity check at all.

Two properties nobody had recorded: **pmat follows `include!`**, so analysing `forward_utils.rs` reports a violation in `logits.rs`; and when pmat cannot produce a report, the `grep -qE 'Errors: *[1-9]'` simply does not match, so **the old gate silently passed**.

Now gated on the **delta** against HEAD. Measured both directions on the real files:

```
gemm.rs / forward_utils.rs / gemv_dispatch.rs / apply.rs   OLD rc=1 -> NEW rc=0
deepening cublas_prefill_gemm   REFUSED: WORSE ... = 40 (was 30 at HEAD)
```

**The live hook was deliberately not installed.** It is untracked, shared across every worktree on this box, and other agents are committing right now — run `make hooks-install`. Case tables found two defects review would not have: an unanchored `grep -n | cut` exited under `set -e`+`pipefail` so the installer failed **mutely**, and `END_MARK_RE` matched two lines because the no-source-files fast path prints the same marker ~50 lines earlier.

## PERF-052 — the ungated ratchet made a new defect unrecordable (#2757)

G2.3 ratcheted the **total** `broken-and-ungated` count, so recording a 45th `quality<=4` ungated row failed **even with a triage entry**. Under time pressure the cheapest way to green was to not record the finding, or to score it above 4 — a ratchet that makes honest reporting the most expensive option.

Reproduced against main before changing anything:

```
G2.4 waivers  PASS  45 broken-and-ungated, 45 triaged, 1 new
G2.3 floors   FAIL: broken-and-ungated is 45, must be <= 44
```

Now ratchets `untriaged_broken()`, floored from the **comparand's** ledger, not the working tree.

**Removing the total ratchet is not a net weakening, and that was tested rather than asserted.** The total blocked three moves; only one was a real regression — a gated row *losing* its gate while broken. That is now `G2.3 gate-loss`, keyed **per row**, because a count cannot express it: moving a gate off a broken row onto a healthy peer in the same cluster and binary leaves covered-total, per-binary and per-cluster counts all untouched. Verified on the real 830-row ledger, where that count-neutral swap goes RED and deleting the new rule makes it GREEN.

G2.1 was left alone deliberately. A cheap version of its citation check — verifying the cited line merely exists and is non-blank — **passes all ten of the stale rows** found earlier in this epic, and a check that launders stale rows is worse than none.

## PERF-042 / PERF-043 — pin-key and clippy-scope guards

`check_pin_keys_consumed.sh` (35 declared keys, 27 consumed, 8 ledgered on a two-way ratchet) and `check_gate_scope_not_narrowed.sh` + `lib/gate_scope.py` with 10 fixtures. `feat/v1-batchred` was **excluded** after confirming it is already an ancestor of `feat/v9-batchfix` (#2771), so nothing duplicates.

## The integration defect only the batch could see

`check_complexity_gate_wiring.sh` arrived named by no workflow — v12 left wiring out of scope — and `check_guards_are_wired.sh` refused it. Each unit is green alone; only the batch puts a new guard in front of the guard that governs guards. Two steps added to `ci.yml`: the wiring guard, and `make hooks-delta-test` for its 66 case-table rows.

## Verification (combined tree)

`check_guards_are_wired`, `check_dogfood_coverage` (+self-test), `check_baseline_ratchets`, `check_no_claim_literals`, `check_mutation_registry`, `check_roadmap_completion_is_cited`, `perf_gate --selftest`, `check_workflow_env_defined`, `check_runner_labels`, `check_complexity_gate_wiring`, `make hooks-delta-test` — all rc=0.

Both gate changes **loosen** a gate, so each was independently verified to still go RED: 8 constructions for the delta gate (including new-file, new-function-in-violating-file, and rename-and-worsen) and 5 for the triage ratchet (including a count-neutral gate-loss swap and a rename evasion) — all built against fresh fixtures rather than replaying the authors' own cases.

`scripts/perf-matrix.yaml` untouched; all 8 cells remain `UNMEASURED`.

### CODE REVIEW

**Comment by noahgift** [comment]
PP-066 triage (epic #2873, ticket #2986): **MERGE-0.66:I-24 → STOP:conflict** — one rebase attempt onto `origin/main` @ cdc0acb99 (`git rebase origin/main` in a detached worktree) stopped on: `scripts/check_bench_protocol.sh scripts/check_comparator_flags.sh scripts/lib/parity_block.py scripts/llama_bin.sh scripts/llama_pin.toml scripts/parity_host_receipt.sh`. Aborted (`git rebase --abort`), branch untouched, labelled `pp-066/merge,pp-066/conflict`. Batch; gate was GREEN on the stale base (11 pass), so not split. The first conflict (.gitignore, both sides appended) was resolved as a union and the replay continued; the second stop is the comparator-pin/parity-block cluster, which main reworked. Linked issues map to G-8 (#2757) and S-2 (#2766); intersects I-24 (parity_block.py), C0-1/G-4 (ci.yml). Producer is never the gate: nothing armed.

**Comment by noahgift** [comment]
PP-066 triage (epic #2873, ticket #2986) — **PRIOR-ART CLOSE** (operator decision, STEP 2b). The branch is retained as `prior-art/2773` and is never deleted; the intent is carried on the issue named below. Zero semantic merges were made.

`prior-art: prior-art/2773@e9314a707 · intent → G-8 (#2898; holds #2757) — also S-2 #2922 (#2766) · salvage: scripts/lib/gate_scope_cases/*.{json,yml} (10) scripts/tests/complexity_delta_gate_cases.sh scripts/tests/complexity_delta_gate_install_cases.sh scripts/tests/complexity_delta_gate_regex_cases.sh scripts/tests/fixtures/pmat_generated_pre_commit.fixture · conflicts: scripts/check_bench_protocol.sh scripts/check_comparator_flags.sh scripts/lib/parity_block.py scripts/llama_bin.sh scripts/llama_pin.toml scripts/parity_host_receipt.sh (.gitignore was both-appended and unioned before the replay stopped)`
